tests/e2e/kubetest2-kops/aws/s3.go

```diff
@@ -136,13 +152,53 @@ func (c Client) EnsureS3Bucket(ctx context.Context, bucketName string, publicRea
 // DeleteS3Bucket deletes a S3 bucket with the given name.
 func (c Client) DeleteS3Bucket(ctx context.Context, bucketName string) error {
 	bucketName = strings.TrimPrefix(bucketName, "s3://")
+
+	// Empty the bucket first
+	paginator := s3.NewListObjectsV2Paginator(c.s3Client, &s3.ListObjectsV2Input{
+		Bucket: aws.String(bucketName),
+	})
```

Comment:
It is a good idea, but I would expect that kOps removes all the files from the bucket, otherwise it's a bug.
This approach may hide such issues.

Reply:
Let's remove it for now, please. Thanks!